src/app/module-load/module-load.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { ModelsService } from '../custom-common/models.service';
+@Component({
+  selector: 'app-module-load',
+  templateUrl: './module-load.component.html',
+  styleUrls: ['./module-load.component.sass'],
+})
+export class ModuleLoadComponent implements OnInit {
+  constructor(readonly models: ModelsService, private router: Router) {}
+
+  fileName = '';
+
+  ngOnInit(): void {}
+
+  upload(event: any) {
```

Comment:
upload event function _should_ open the navbar so we can understand what happened. Would be neat if we flashed the new option -- (change background to primary color and back again?)